libs/data-mapper/src/lib/utils/ReactFlow.Util.ts

```diff
@@ -81,6 +82,7 @@ export const useLayout = (
         })
         .catch((error) => {
           console.error(`Elk Layout Error: ${error}`);
```

Comment:
NIT: console.log over here

Reply:
Yeah that one's intentional - any console.errors should be intentional and formatted to help when dev-ing